src/compas/utilities/colors.py

```diff
@@ -64,6 +64,9 @@
 def i_to_rgb(i, normalize=False):
     """Convert a number between 0.0 and 1.0 to an equivalent RGB tuple.
 
+    .. deprecated:: 1.14
```

Comment:
I like this deprecation notice in the docs. However, all the other ones that were recently added to the docs were done just programmatically using `warnings` module. Should we do both everywhere?

Reply:
i will add both everywhere in one big clean up docstring PR across all modules